src/server/req_signal.c

```diff
@@ -104,7 +104,7 @@ req_signaljob(struct batch_request *preq)
 	int		  j;
 	char		 *jid;
 	int		  jt;		/* job type */
-	int		  offset;
+	int       js;		/* job state */
```

Comment:
Format this line so as to align with its previous one.

Reply:
Fixed in rc 16 https://github.com/PBSPro/pbspro/pull/590/commits/bc13b0be5a010f69357b3cde4fccafea0e0fb92d